src/components/navbar/RightBlock.tsx

```diff
@@ -83,42 +90,64 @@ const AccountContainer = ({
   )
 }
 
-export default function ({
+const SuspendedAccount = ({
   account,
   needNetworkChange,
   eNSName,
-}: {
-  account?: string
-  needNetworkChange: boolean
-  eNSName?: string
-}) {
+}: AccountProps) => {
   return (
-    <div className={walletContainer}>
-      <SocialLinks />
-      <SealVerse />
-      <LastDelimiter />
-
-      <Suspense
-        fallback={
-          <div className={accountLinkContainer}>
-            <div className={walletAccount}>
-              <AccentText small color="text-primary-semi-dimmed">
-                <span className={displayTo('lg')}>Fetching...</span>
-                <span className={displayFrom('lg')}>Fetching account...</span>
-              </AccentText>
-            </div>
-            <div className={width('w-fit')}>
-              <Logo connected={false} />
-            </div>
+    <Suspense
+      fallback={
+        <div className={accountLinkContainer}>
+          <div className={walletAccount}>
+            <AccentText small color="text-primary-semi-dimmed">
+              <span className={displayTo('lg')}>Fetching...</span>
+              <span className={displayFrom('lg')}>Fetching account...</span>
+            </AccentText>
           </div>
-        }
-      >
-        <AccountContainer
-          eNSName={eNSName}
+          <div className={width('w-fit')}>
+            <Logo connected={false} />
+          </div>
+        </div>
+      }
+    >
+      <AccountContainer
+        eNSName={eNSName}
+        needNetworkChange={needNetworkChange}
+        account={account}
+      />
+    </Suspense>
+  )
+}
+
+export default function ({
+  account,
+  needNetworkChange,
+  eNSName,
+}: AccountProps) {
+  return (
+    <>
+      <div className={walletContainer}>
+        <SocialLinks />
+        <SealVerse />
+        <LastDelimiter />
+        <SuspendedAccount
+          account={account}
           needNetworkChange={needNetworkChange}
+          eNSName={eNSName}
+        />
+      </div>
+
+      <div className={displayTo('xs')}>
+        <SuspendedAccount
           account={account}
+          needNetworkChange={needNetworkChange}
+          eNSName={eNSName}
         />
-      </Suspense>
-    </div>
+      </div>
+      <div className={displayTo('xs')}>
+        <SealVerse />
+      </div>
```

Comment:
Can we put `SuspendedAccount & SealVerse` into one `div` element here, please?

Reply:
No, we need to keep them separate, because the parent element will apply some styles for them, they should be a separate divs
![image](https://user-images.githubusercontent.com/49658988/194227407-2b4355a9-1e28-4aa4-bfdf-fcafe49fb6da.png)
![image](https://user-images.githubusercontent.com/49658988/194227467-226d6cba-2769-46e3-8388-f48814f89244.png)